tcases-lib/src/main/java/org/cornutum/tcases/DefUtils.java

```diff
@@ -77,5 +100,6 @@ public static void assertPropertyIdentifiers( Collection<String> properties) thr
     }
 
   private static final Pattern identifierRegex_ = Pattern.compile( "[\\w\\-]+");
+  private static final Pattern varValueRegex_ = Pattern.compile( "[^\\p{Cntrl}]*");
```

Comment:
avoiding chars like newline, escape, bell. Those could be encoded in XML1.1, but the SAX parser fails to recognize them.